project_national/fact_sheets_category_1.yml

```diff
@@ -0,0 +1,465 @@
+        apply_logic:
+          - or: 
+            - HVAC Cooling Efficiency|Room AC, EER 8.5
+            - HVAC Cooling Efficiency|Room AC, EER 9.8
+            - HVAC Cooling Efficiency|Room AC, EER 10.7
+          - HVAC Cooling Type|Room AC
+
+      # Heat Pumps
+      # SEER 14 for northern states and SEER 15 for southern states, comes with HSPF == 8.8 which is the federal minimum
+      # southern states
+      - *heat_pump_fed_min_ducted_southern
+      - *heat_pump_fed_min_ductless_southern
+      - *heat_pump_fed_min_ducted_northern
+      - *heat_pump_fed_min_ductless_northern
+      #global
+      - &heat_pump_ducted_cooling
+        option: HVAC Cooling Efficiency|Ducted Heat Pump # Applied regardless of which option is applied
+        apply_logic:
+          or:
+            - *logic_fed_min_heat_pump_ASHP_ducted_northern
+            - *logic_fed_min_heat_pump_ASHP_ducted_southern
+
+      - &heat_pump_ductless_cooling
+        option: HVAC Cooling Efficiency|Non-Ducted Heat Pump # Applied regardless of which option is applied
+        apply_logic:
+          or:
+            - *logic_fed_min_heat_pump_MSHP_ductless_northern
+            - *logic_fed_min_heat_pump_MSHP_ductless_southern
+
+  - upgrade_name: Federal Minimum Heating and Cooling
+    options:
+    ####
+    # Federal minimum heat pumps
+      - *heat_pump_fed_min_ducted_southern
+      - *heat_pump_fed_min_ductless_southern
+      - *heat_pump_fed_min_ducted_northern
+      - *heat_pump_fed_min_ductless_northern
+      - *heat_pump_ducted_cooling
+      - *heat_pump_ductless_cooling
+
+    # Federal minimum AC - central and room
+      - *room_AC_fed_min
+      - *central_AC_fed_min_southern
+      - *central_AC_fed_min_northern
+
+    # Federal minimum boilers - NG and fuel oil
+      - *natural_gas_boiler_fed_min
+      - *fuel_oil_boiler_fed_min
+
+    # Federal minimum furnaces - NG and fuel oil
+      - *natural_gas_furnace_fed_min_north
+      - *natural_gas_furnace_fed_min_south
+      - *fuel_oil_furnace_fed_min
+
+  - upgrade_name: Federal Minimum Water Heating
+    options:
+    #### Electric water heater - applies to electric water heaters with less efficiency. Upgrade is 3.3 UEF HPWH ####
+      - &HPWH_fed_min_30_gal
+        option: Water Heater Efficiency|Electric Heat Pump, 30 gal, 3.3 UEF
+        apply_logic:
+          - Water Heater Fuel|Electricity
+          - or:
+            - Bedrooms|1
+            - Bedrooms|2
+
+      - &HPWH_fed_min_50_gal
+        option: Water Heater Efficiency|Electric Heat Pump, 50 gal, 3.3 UEF
+        apply_logic:
+          - Water Heater Fuel|Electricity
+          - or:
+            - Bedrooms|3
+            - Bedrooms|4
+
+      - &HPWH_fed_min_66_gal
+        option: Water Heater Efficiency|Electric Heat Pump, 66 gal, 3.3 UEF
+        apply_logic:
+          - Water Heater Fuel|Electricity
+          - Bedrooms|5
+
+    #### Gas storage
```

Comment:
NIT, `gas_wh_fed_min_30_gal` and `gas_wh_fed_min_40_gal` can be combined

Reply:
For my sanity's sake, could you combine these two anchors into one? The tank is auto-sized according to [Table 8 of 2014 BAHSP](https://www.energy.gov/sites/prod/files/2014/03/f13/house_simulation_protocols_2014.pdf), which depends on both bedroom and bathroom counts. e.g., 3bed+1bath is assigned a 30 gal tank while 4bed+3bath is assigned 50 gal. @whiphi92